### PR TITLE
Armor scaling sanity rebalance

### PR DESCRIPTION
**Pull request by albertbokor**
R&G breaks bipods and some other things and just generally doesn't work and isnt maintained, pls get rid of it already!

Changed armor scaling to values proposed by Sam here https://discord.com/channels/278818534069501953/356498503847116801/1064071382952067092
The scale of armor value change is very annoying at lower values making preindustrial armors hard to relative-balance against eachother. These values are bit higher than I had in mind but still much better than before.

### CODE REVIEW

**Review by N7Huntsman — COMMENTED** [review]
**Run & Gun:**

**Armor Scaling:**
From what I'm gleaning from the brief discord conversation (and correct me if I'm wrong), the intention this PR is trying to address the balance of AP vs Armor for basic melee weapons--medieval ones, in this context.

However, with that being the intention, there are some things in this PR I don't understand. My full reasoning is below, I'll provide a quick summary here:

**TL;DR**
- I would adjust the `ArmorRatingBase` multipliers for Awful, Poor, and Masterwork to match vanilla.
- I would do nothing else.
- Given the values involved, the suggested multiplier changes do a lot to impact late-game melee weapons/armor and very little to help early melee weapons/armor.
- There are other mechanics/systems (criticals, partial armor, etc.) that I think better address this.
- The suggested health-multiplier changes would make armor stronger quite a bit stronger against simple melee weapons.

**MeleePenetrationFactor**
It makes sense to raise the `MeleePenetrationFactor` for the lower qualities (though given the small baseline AP values, I don't think this is going to impact much). However, I don't see why we're reducing the factor for the higher qualities as well--except for "balancing," but I don't think that's going in the right direction here.

Moreover, the quality distribution of weapons NPCs are spawning with (if a player is using awful melee weapons for any kind of serious fighting, they have no one to blame but themselves) and the fact that some weapons (like clubs) don't have quality is going to further reduce the impact of this--for as little as the changes are going to help basic melee weapons and will have a much larger impact on more advanced weapons, I'd leave this alone.

If you feel like you _have_ to do something, you could increase the `Awful` to 0.6 or 0.65--but I certainly wouldn't reduce with the higher quality stat multipliers

**ArmorRatingBase**
In the same vein as above, there are some odd choices if the overall intention is to make the melee AP more competitive against early armors--some of which is going to have a much greater impact on higher-tech armor than early stuff.

It's worth noting that for the higher qualities, CE already uses a more conservative scale than vanilla. Obviously, it's not a like-to-like comparison given vanilla different armor system but here's the vanilla quality scale for comparison:

```
        <factorAwful>0.60</factorAwful>
        <factorPoor>0.80</factorPoor>
        <factorNormal>1.00</factorNormal>
        <factorGood>1.15</factorGood>
        <factorExcellent>1.30</factorExcellent>
        <factorMasterwork>1.45</factorMasterwork>
        <factorLegendary>1.80</factorLegendary>
```
I would say that we could adjust CE's multipliers for Awful, Poor, and Masterwork to match the vanilla value--that would improve the lowest qualities. I wouldn't make any of these other suggested changes, because I don't think they make the changes we're hoping to see with this PR.

It's also worth noting that in vanilla, armor condition doesn't actually affect protection. In CE, condition affects armor protection but not melee penetration--that means you can beat their armor down to a level you can penetration. Given the importance of being able to do this to defeat heavier worn armor with melee, I would not raise the condition-protection baseline multiplier from 0 to 0.25--this would be a move in the opposite direction.

In terms to the broader topic of "Melee vs Industrial-tech Armor," it's probably not ideal that bullets and blades both work off of sharp armor (though, for the record, I do **not** ever see CE implementing a separate "ballistic" armor type--it's a non-starter on the compatibility side.) That said, so long as we're tying our penetration values to physics and RHA (and I don't see a reason to change that), there's always going to be a gulf between the armor thickness melee can penetrate and the armor thickness needed to protect against gunfire.

From the melee side, I chiefly see our partial armor coverage, critical hit system, and melee part targeting mechanics as our chief means to address that disparity--it's a lot easier to go around the armor than trying to get melee to go through it.

There's not a lot of room to address that by adjusting quality multipliers, and we're going to see an outsized impact on stuff with higher armor values like power armor--where the stat multiplier is going to be more meaningful to the actual protection of the armor--while trying to make it matter for low level armor like plate armor.

**Comment by albertbokor** [comment]
My angle of approach was from an "xml stats writer" side, idk, just pissed me off how much different a lot of items as ingame generated from their baseline stats, may or may not be an ctual issue, sure, but was rather frustrating when i had to squeeze in several preindustrial armors into a progression. 
I still consider the deviation too much, in practice its pretty much just free buff to player.

**Comment by albertbokor** [comment]
task "succeded" unsuccesfully...
